index.html

```diff
@@ -88,25 +91,143 @@
+									</span>
+								</li>
+								<li>
+									<a href="UmbracoCms.4.8.1.zip" rel="download">4.8.1</a>
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.8.1" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									<a href="UmbracoCms.4.9.0.zip" rel="download">4.9.0</a>
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.9.0" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									<a href="UmbracoCms.4.9.1.zip" rel="download">4.9.1</a>
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.9.1" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.10.0
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.10.0" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
 								<li><a href="UmbracoCms.4.10.1.zip" rel="download">4.10.1</a> - introduced native MVC (<a href="http://umbraco.com/follow-us/blog-archive/2012/11/14/security-update-for-4100.aspx" rel="external" title="Security update for 4.10.0">v4.10.0 had a short release</a>)</li>
+								<li>
+									4.11.0
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.11.0" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.11.1
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.11.1" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.11.2
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.11.2" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.11.2.1
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.11.2.1" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.11.2.2
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.11.2.2" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.11.3
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.11.3" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>

```

Comment:
I'd remove these releases, (between v4.11.1-3), I remember at the time they came out so close together that the v4.11.3.1 wraps it all up.